src/ui/public/index_patterns/_index_pattern.js

```diff
@@ -176,15 +176,16 @@ define(function (require) {
         return this.intervalName && _.find(intervals, { name: this.intervalName });
       };
 
-      self.toIndexList = function (start, stop) {
+      self.toIndexList = function (start, stop, sortDirection) {
         return new Promise(function (resolve) {
           var indexList;
           var interval = self.getInterval();
 
           if (interval) {
             indexList = intervals.toIndexList(self.id, interval, start, stop);
+            if (sortDirection === 'desc') sortDirection = sortDirection.reverse();
```

Comment:
I think you want to `reverse` the `indexList` rather than the `sortDirection` string:

```js
if (sortDirection === 'desc') indexList = indexList.reverse();
```